### PR TITLE
Revise descriptions in Portal about waiver and confirmation

### LINKED ISSUE #249: [Portal] Waiver description needs elaboration
For full clarity, we can describe how the waiver button will take users to DocuSign and then instruct users to return to the Portal to confirm their attendance.

### PR DESCRIPTION
**Pull request by taesungh**
Fixes #249.
- Describe how waiver will be handled through DocuSign
- Ask users to return to Portal after completing waiver
- Extract and restyle `<hr>` with styles from Reboot

### CODE REVIEW

File: apps/site/src/app/(main)/portal/@applicant/ApplicantPortal.tsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ async function Portal() {
 			<h2 className="text-4xl font-semibold">Status</h2>
 			<VerticalTimeline status={status as PortalStatus} />
 			<Message status={status as PortalStatus} />
+			{isAccepted && <hr />}
 			{isAccepted && <SignWaiver status={status as PortalStatus} />}
```
Comment (samderanova) [inline apps/site/src/app/(main)/portal/@applicant/ApplicantPortal.tsx:38]: Suggestion: Maybe combine these two if they have the same conditional?

File: apps/site/src/app/(main)/portal/@applicant/SignWaiver.tsx [hunk 2]
```diff
@@ -9,9 +9,12 @@ interface SignWaiverProps {
 function SignWaiver({ status }: SignWaiverProps) {
 	return status === PortalStatus.waived ? null : (
 		<div>
-			<hr />
 			<p>
-				In order to attend IrvineHacks 2024, please fill out the waiver below
+				In order to attend IrvineHacks 2024, all participants must complete the
+				Participation Waiver and review the Code of Conduct. By using the button
```
Comment (samderanova) [inline apps/site/src/app/(main)/portal/@applicant/SignWaiver.tsx:14]: Suggestion: "By clicking the button" would be a bit more appropriate here

**Review by samderanova — CHANGES_REQUESTED** [review]

File: apps/site/src/app/(main)/portal/@applicant/ApplicantPortal.tsx (hunk 1 above)
Comment (taesungh) [reply apps/site/src/app/(main)/portal/@applicant/ApplicantPortal.tsx:38]: I'll need to reconsider based on the other conditional introduced in #252.

File: apps/site/src/app/(main)/portal/@applicant/ConfirmAttendance.tsx
Comment (rileysw) [inline apps/site/src/app/(main)/portal/@applicant/ConfirmAttendance.tsx:1]: For waitlist, change the last sentence to

"Please check your email for more info about the waitlist and waitlist walk-ins!"

**Review by rileysw — CHANGES_REQUESTED** [review]

File: apps/site/src/app/(main)/portal/@applicant/Message.tsx
Comment (rileysw) [inline apps/site/src/app/(main)/portal/@applicant/Message.tsx:1]: For waitlist, change the last sentence to

"Please check your email for more info about the waitlist and waitlist walk-ins!"

**Review by rileysw — CHANGES_REQUESTED** [review]

File: apps/site/src/app/(main)/portal/@applicant/SignWaiver.tsx [hunk 3]
```diff
@@ -3,10 +3,12 @@ import Button from "@/lib/components/Button/Button";
 function SignWaiver() {
 	return (
 		<div>
-			<hr />
 			<h3 className="text-3xl my-4">Waiver</h3>
-			<p className="mt-4">
-				In order to attend IrvineHacks 2024, please fill out the waiver below
+			<p>
+				In order to attend IrvineHacks 2024, all participants must complete the
+				Participation Waiver and review the Code of Conduct. The button below
```
[the remark this answers is not in the document]
Comment (samderanova) [reply apps/site/src/app/(main)/portal/@applicant/SignWaiver.tsx:9]: Is this still needed when the DocuSign PowerForm already includes the code of conduct along with the waiver?

File: apps/site/src/app/(main)/portal/@applicant/Message.tsx
Comment (taesungh) [reply apps/site/src/app/(main)/portal/@applicant/Message.tsx:1]: Updated in f53cc42.

**Review by rileysw — APPROVED** [review]

**Review by samderanova — APPROVED** [review]

**Pull request merged**